packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1098,6 +1099,7 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
     WidgetsBinding.instance!.removeObserver(this);
     _stopListeningToStream();
     _scrollAwareContext.dispose();
+    _replaceImage();
```

Comment:
The code here would be a little clearer with a named arg set to null to indicate that the image is getting replaced by nothing?

Reply:
Done